builtin/credential/github/path_login.go

```diff
@@ -159,9 +158,22 @@ func (b *backend) verifyCredentials(ctx context.Context, req *logical.Request, t
 		}
 	}
 
-	if config.Organization == "" {
-		return nil, logical.ErrorResponse(
-			"organization not found in configuration"), nil
+	if config.OrganizationID == 0 {
+		// Previously we did not verify using the Org ID. So if the Org ID is
+		// not set, we will trust-on-first-use and set it now.
+		err = config.setOrganizationID(ctx, b)
```

Comment:
If `setOrganizationID` accepts a client instead, we could pass in the client with the login token from [L179](https://github.com/hashicorp/vault/pull/13332/files#diff-67438541539dbbb356d23f4b60a8fb78c4a89ed1c3f4cb51a0e9fa8044f8ccaeR179). I'm not sure if that token requires extra permission to query the org ID though.